src/Microsoft.DotNet.Wpf/specs/AutomationUpdate.md

```diff
@@ -0,0 +1,503 @@
+Automation update
+==
+
+# Background
```

Comment:
It would be helpful for this spec and then hopefully propagate into the See Also of the docs to link to what we're wrapping and prior art, primarily
* [UiaRaiseActiveTextPositionChanged](https://docs.microsoft.com/en-us/windows/win32/api/uiautomationcoreapi/nf-uiautomationcoreapi-uiaraiseactivetextpositionchangedevent).
* [AutomationPeer.RaiseNotificationEvent](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Automation.Peers.AutomationPeer.RaiseNotificationEvent)
* [AutomationPeer.GetHeadingLevel](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Automation.Peers.AutomationPeer.GetHeadingLevel)
* [AutomationPeer.IsDialog](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Automation.Peers.AutomationPeer.IsDialog)

Reply:
For the spec review, your links should be good enough.

The See Also of docs for WPF automation properties don't link to anything outside of .NET (just to some overview pages);  breaking that pattern now would be weird, IMO.